src/OptiHPLCHandler/applications/empower_implementation/empower_tools.py

```diff
@@ -100,32 +100,64 @@ def determine_last_high_flow_time(gradient_table: List[dict]) -> float:
 
 def determine_strong_eluent(gradient_table: List[dict]) -> Optional[str]:
     """
-    Determines the strong eluent from a given gradient table.
+    Determine the strong eluent in the gradient table. Assuming there is only one
+    strong eluent. Deprecated in favor of classify_eluents.
+    """
+    classified_eluents_dict = classify_eluents(gradient_table)
+    list_strong_eluent = classified_eluents_dict["strong_eluents"]
+    list_weak_eluents = classified_eluents_dict["weak_eluents"]
+    list_constant = classified_eluents_dict["constant_composition_eluents"]
+    return list_strong_eluent[0], [*list_weak_eluents, *list_constant]
 
-    Args:
-        gradient_table (List[dict]): A list of dictionaries representing the gradient
-        table.
 
-    Returns:
-        Tuple[str, List[str]]: A tuple containing the strong eluent and a list of weak
-        eluents.
+def determine_decreasing_weak_eluents(gradient_table: List[dict]) -> List[str]:
+    """
+    Determine the weak eluents in the gradient table. Deprecated in favor of
+    classify_eluents.
+    """
+    classified_eluents_dict = classify_eluents(gradient_table)
+    list_weak_eluents = classified_eluents_dict["weak_eluents"]
+
+    return list_weak_eluents
+
+
+def classify_eluents(
+    gradient_table: List[dict],
+) -> dict[List[str], List[str], List[str]]:
```

Comment:
This is a 
```suggestion
) -> Dict[str, List[str]]:
```
And rember to import Dict from typing